src/test/java/nl/tudelft/ewi/devhub/server/backend/Bootstrapper.java

```diff
@@ -297,6 +297,16 @@ private void prepareGroup(Map<String, User> userMapping, CourseEdition entity, M
 
 			deliveries.persist(deliveryEntity);
 
+			AssignedTA assignedTA = new AssignedTA();
+			assignedTA.setAssignment(deliveryEntity.getAssignment());
+			assignedTA.setGroup(groupEntity);
+			if(deliveryCounter % 2 == 0) {
+				assignedTA.setTeachingAssistant(userMapping.get("assistant1"));
+				entityManager.persist(assignedTA);
+			} else {
+				//assignedTA.setTeachingAssistant(userMapping.get("assistant2"));
```

Comment:
Add a `TODO` here to indicate this still needs to be fixed (or fix it directly in this PR)